### DISCUSSION TITLE
Add option to replace default css class and variable prefix

### DISCUSSION THREAD

**Discussion by harishpanneerselvam** [mantinedev/mantine#1488] [Feature requests]
Currently, it only supported in emotionOptions

**Comment by moisreis** [comment]
How do I remove the "mantine-" prefix on my classes?

**Reply by rtivital** [reply]
https://mantine.dev/theming/emotion-cache/

**Reply by GuyShaanan** [reply]
(That link isn't working anymore.)

Joining the discussion, it will be awesome to be able to change the css variables prefix to suit our brand naming, from `--mantine-radius-md` to, lets say, `--mycompany-radius-md`.

[MUI lets you do that](https://mui.com/joy-ui/customization/using-css-variables/#custom-prefixes), and I know Mantine offers [classNamePrefix](https://mantine.dev/theming/mantine-provider/#classnamesprefix) but it only works for the static classes.

Somewhat related, for Input with labels, Mantine auto assigns IDs with `mantine` prefix. We can set our own IDs, but it would be great to have the ability to set a default custom prefix for these as well.

**Reply by rtivital** [reply]
It is not possible, Mantine provides static CSS.
